src/Summoner/Template.hs

```diff
@@ -109,6 +111,37 @@ createStackTemplate ProjectData{..} = Dir (toString repo) $
         $endLine
         |]
 
+    ghcOptions :: Text
+    ghcOptions = case warnings of
+        [] -> defaultWarnings
+        xs -> T.intercalate "\n" xs
+
+    defaultWarnings :: Text
+    defaultWarnings =
+        [text|
+        -Wincomplete-uni-patterns
+        -Wincomplete-record-updates
+        -Wmissing-import-lists
+        -Wcompat
+        -Widentities
+        $versionWarnings
+        |]
+
+    hasLeast :: Ord a => [a] -> a -> Bool
+    hasLeast list el = list == filter (>= el) list
```

Comment:
The implementation of this function is wrong. If I call this function like this:

```haskell
hasAtLeast [Ghc7103, Ghc843] Ghc822
```

then it will return `False` while it should return `True`. `list` is `[Ghc7103, Ghc843]` and `filter (>= el) list` is `[Ghc843]`. Obviously, `[Ghc7103, Ghc843] /= [Ghc843]` and that's why we get `False`.